src/index.css

```diff
@@ -69,38 +56,559 @@ a {
-  flex-direction: row;
-  justify-content: space-between;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem 0 3rem 0;
+}
+
+.addBook {
+  min-width: 10vw;
+  height: 2.5rem;
+  border-radius: 4px;
+  font-family: 'Roboto Slab', serif;
+  background-color: var(--primary-color);
+  outline: none;
+  border: solid 1px var(--white-three);
+  color: white;
+}
+
+/* mobile version */
+@media (width >= 765px) {
+  .navItems {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 5%;
+  }
+
+  .panel-bg {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    min-height: 5.938rem;
+    background-color: white;
+    position: sticky;
+    padding: 0 2.5rem 0 2.5rem;
+    top: 0;
+    overflow-x: hidden;
+  }
+
+  .bookstore-CMS {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 32px;
+    font-weight: 600;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: #0290ff;
+  }
+
+  .books-nav {
+    color: #121212;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 17px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: 1.9px;
+  }
+
+  .pageWrapper {
+    margin: auto;
+    width: 80vw;
+    background-color: #fafafa;
+    border-radius: 4px;
+    box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.2);
+  }
+
+  .innerWrapper {
+    width: 90%;
+    margin: auto;
+  }
+
+  .bookPanelBg {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    margin-top: 4.375rem;
+    gap: 2rem;
+  }
+
+  .bookPanel {
+    min-height: 11rem;
+    padding: 1rem 2.5rem 1rem 2.5rem;
+    border-radius: 4px;
+    border: solid 1px var(--white-three);
+    background-color: var(--white-two);
+    align-items: center;
+  }
+
+  .bookDetails {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .firstCol {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 1.5rem;
+  }
+
+  .title {
+    font-family: 'Roboto Slab', serif;
+    font-size: 30px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: -0.2px;
+    color: var(--black-two);
+  }
+
+  .author {
+    font-family: 'Roboto Slab', serif;
+    font-size: 17px;
+    font-weight: 300;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: #4386bf;
+  }
+
+  .category {
+    width: 7rem;
+    height: 1.125rem;
+    margin: 0 4rem 0 0;
+    opacity: 0.5;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 0.875rem;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
```

Comment:
- [ ] The add book section should stick to the Zeplin design , Kinldy fix that 👍 

### Your design
![Capture](https://github.com/Rafiduzzaman/bookstore/assets/80458199/37bb7b5c-8a94-4374-9d96-ce238403227f)

### Zeplin design
![Capture0](https://github.com/Rafiduzzaman/bookstore/assets/80458199/8130e98e-b557-48ef-9234-2452e26906cd)